src/Model.js

```diff
@@ -265,7 +276,10 @@ export default class Model {
                 if (RelModel.prototype instanceof Store) {
                     return new RelModel(options);
                 }
-                return new RelModel(null, options);
+                // If we have a related model, we want to force the related model to have id null as that means there is no model set
+                const newModelData = {}
+                newModelData[RelModel.primaryKey] = null;
+                return new RelModel(newModelData, options);
```

Comment:
Note: you can use expressions as keys in js by putting  them between square brackets. So this could be:
```js
return new RelModel({ [RelModel.primaryKey]: null }, options);
```

Reply:
See latest commit